src/components/Card/Сard.tsx

```diff
@@ -0,0 +1,8 @@
+import { HTMLAttributes, PropsWithChildren, ReactElement } from 'react';
+import styles from './Card.module.css';
+
+export function Card({ children, className, ...rest }: PropsWithChildren<HTMLAttributes<HTMLDivElement>>): ReactElement {
+	return <div {...rest} className={`${className} ${styles.card}`}>
```

Comment:
I don't think you need to allow to pass all the div`s props (I mean `rest` operator). It's makes component too unpredictable. Also `classnames` library suits fine here to set classes

Reply:
Yes, make sense